uikit/src/main/java/com/midtrans/sdk/uikit/views/uob/web/UobWebPaymentActivity.java

```diff
@@ -0,0 +1,202 @@
+package com.midtrans.sdk.uikit.views.uob.web;
+
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.View;
+import android.view.ViewStub;
+import android.widget.Toast;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AlertDialog;
+
+import com.midtrans.sdk.corekit.core.Logger;
+import com.midtrans.sdk.corekit.models.TransactionResponse;
+import com.midtrans.sdk.uikit.R;
+import com.midtrans.sdk.uikit.abstracts.BasePaymentActivity;
+import com.midtrans.sdk.uikit.utilities.SdkUIFlowUtil;
+import com.midtrans.sdk.uikit.utilities.UiKitConstants;
+import com.midtrans.sdk.uikit.widgets.FancyButton;
+
+public class UobWebPaymentActivity extends BasePaymentActivity implements UobWebPaymentView {
+
+    private UobWebPaymentPresenter presenter;
+    private FancyButton buttonPrimary;
+    private Boolean isAlreadyGotResponse = false;
+    private int uobEzpayIntentCode, attempt;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        showProgressLayout();
+        setContentView(R.layout.uikit_activity_uob_web_payment);
+        initPresenter();
+        initLayout();
+        initData();
+        initActionButton();
+        hideProgressLayout();
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+
+        if (uobEzpayIntentCode == UiKitConstants.INTENT_CODE_UOBEZPAY && presenter != null) {
+            presenter.getPaymentStatus();
```

Comment:
do we need check status for UOB WEB ?? i think the status will `Pending` right for UOB web and no need to check status

Reply:
@askur87 if customer click to pay before returning to the app, we also need to show the  PaymentStatusPage right?
![image](https://user-images.githubusercontent.com/8348435/122823741-50004880-d312-11eb-843a-338fe3a7baf8.png)

Reply:
hmm... for UOB web I think we don't need to show paymentStatus page just return the status as pending, you can stimulate using e-Pay BRI, the flow should be same